### ISSUE TITLE
id and anchor not work, when heading not include English or Number

### ISSUE THREAD

**Issue by Airkro** [simov/markdown-viewer#18]
id and anchor not work, when heading not include English or Number.
sidebar jumping not work too.

example:
```markdown
# 测试一
## 测试二
### 测试三
```

output
```html
<div id="html" class="markdown-body">
    <h1 id="-">测试一</h1>
    <h2 id="-">测试二</h2>
    <h3 id="-">测试三</h3>
</div>

<div id="_toc">
    <div id="_ul">
        <a href="#-">测试一</a>
        <div id="_ul">
            <a href="#-">测试二</a>
            <div id="_ul">
                <a href="#-">测试三</a>
            </div>
        </div>
    </div>
</div>
```

**Comment by simov** [comment]
Can you copy/paste some example markdown here?

**Comment by Airkro** [comment]
sorry for my negligence，I have changed the title and comment.

**Comment by simov** [comment]
Thanks! Again this is related to the [marked](https://github.com/chjj/marked) parser that I'm using internally. I'm going to research about other possible markdown parsers and I'll take into account this issue as well.

**Comment by simov** [comment]
I just published version 2.8 where you can choose between different compilers.

For anchors to work you have to choose the `showdown` compiler.

Let me know if you have any other issues.

**Comment by simov** [comment]
Another quick release, I switched to the [Remark](https://github.com/wooorm/remark) compiler. You should receive the update automatically. It fixes this issue.
